app/src/main/res/layout/fragment_qr_profile_after_scan.xml

```diff
@@ -0,0 +1,155 @@
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/quicksand"
+        android:text="XX"
+        android:textAlignment="textStart"
+        android:textColor="@color/main_blue"
+        android:textSize="@dimen/title_size"
+        android:textStyle="bold"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="HardcodedText" />
+
+    <ImageButton
+        android:id="@+id/button_toggle_visibility_on"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:background="@drawable/ic_button_location_on_dark"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <View
+        android:id="@+id/view_divider"
+        android:layout_width="wrap_content"
+        android:layout_height="1dp"
+        android:alpha="0.2"
+        android:background="@color/accent_grey_blue_dark"
+        app:layout_constraintEnd_toStartOf="@id/button_toggle_visibility_on"
+        app:layout_constraintHorizontal_bias="1.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_qr_value" />
+
+    <Space
+        android:id="@+id/qr_profile_space"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/large_space"
+        app:layout_constraintTop_toBottomOf="@id/view_divider" />
+
+    <TextView
+        android:id="@+id/text_lon_lat"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/quicksand"
+        android:text="(Longitude, Latitude)"
+        android:textColor="@color/secondary_blue_dark"
+        android:textSize="@dimen/text_size"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/qr_profile_space"
+        tools:ignore="HardcodedText" />
+
+
+    <TextView
+        android:id="@+id/text_scans"
+        android:layout_width="100dp"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/quicksand"
+        android:text="Scans: XX"
+        android:textAlignment="textEnd"
+        android:textColor="@color/secondary_blue_dark"
+        android:textSize="@dimen/text_size"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/qr_profile_space"
+        tools:ignore="HardcodedText" />
+
+
+    <!-- figure out how make square-->
+    <ImageView
+        android:id="@+id/image_holder"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintDimensionRatio="H,1:1"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        android:background="@drawable/rec_image_holder_bg"
+        app:layout_constraintTop_toBottomOf="@id/text_lon_lat"
```

Comment:
every file here could really use some autoformatting (Ctrl+alt+L)

Reply:
Do I just do Ctrl+alt+L on every line, or just Ctrl+alt+L once in the xml code? Because it says that the content is already properly formatted

Reply:
and I'm assuming the proper format is
android:
app:
tools: